Transfer/Sources/View/TransferTableViewFooterView.swift

```diff
@@ -18,15 +18,15 @@
 
 import UIKit
 
-final class TrasferTableViewFooterView: UITableViewHeaderFooterView {
-    static let reuseIdentifier = "TrasferTableViewFooterViewIdentifier"
+final class TransferTableViewFooterView: UITableViewHeaderFooterView {
+    static let reuseIdentifier = "TransferTableViewFooterViewIdentifier"
 
     lazy var footerLabel: UILabel = {
         let label = UILabel(frame: .zero)
         label.lineBreakMode = .byWordWrapping
         label.numberOfLines = 0
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.accessibilityIdentifier = "TrasferTableViewFooterLabelIdentifier"
+        label.accessibilityIdentifier = "TransferTableViewFooterLabelIdentifier"
```

Comment:
update TransferTableViewFooterLabelIdentifier to transferFooterLabelIdentifier

Reply:
updated